lms/djangoapps/discussion/rest_api/api.py

```diff
@@ -517,17 +526,21 @@ def _serialize_discussion_entities(request, context, discussion_entities, reques
 
 
 def get_thread_list(
-        request,
-        course_key,
-        page,
-        page_size,
-        topic_id_list=None,
-        text_search=None,
-        following=False,
-        view=None,
-        order_by="last_activity_at",
-        order_direction="desc",
-        requested_fields=None,
+    request: Request,
+    course_key: CourseKey,
+    page: int,
+    page_size: int,
+    topic_id_list: List[str] = None,
+    text_search: Optional[str] = None,
+    following: Optional[bool] = False,
+    author: Optional[str] = None,
+    thread_type: Optional[ThreadType] = None,
+    flagged: Optional[bool] = None,
+    view: Optional[ViewType] = None,
+    order_by: ThreadOrderingType = "last_activity_at",
+    order_direction: Literal["desc"] = "desc",
+    requested_fields: Optional[List[Literal["profile_image"]]] = None,
+    count_flagged: bool = None,
```

Comment:
=> `count_flagged: bool = False`

Reply:
I've set it to none because `None` will suppress sending the parameter at all while False will stil send the parameter. So `None` captures the intent better.